core/geometry/src/geometry3d/Matrix3d.ts

```diff
@@ -869,377 +897,201 @@ export class Matrix3d implements BeJSONFunctions {
         break;
+      // Start with FRONT view, ccw rotation by 45 degrees around Y and by 35.264 degrees around X
       case StandardViewIndex.RightIso:
         result = Matrix3d.createRowValues(
           0.707106781186548, 0.70710678118654757, 0.00000000000000000,
           -0.408248290463863, 0.40824829046386302, 0.81649658092772603,
           0.577350269189626, -0.57735026918962573, 0.57735026918962573);
         break;
-      case StandardViewIndex.Top: // no rotation
+      // no rotation
+      case StandardViewIndex.Top:
       default:
         result = Matrix3d.createIdentity(result);
     }
     if (invert)
-      result.transposeInPlace();
+      result.transposeInPlace(); // matrix is rigid so transpose and inverse are the same
     return result;
   }
-  /*
-  // this implementation has problems distinguishing failure (normalize) from small angle.
-  public getAxisAndAngleOfRotation(): { axis: Vector3d, angle: Angle, error: boolean } {
-
-    const result = { axis: Vector3d.unitZ(), angle: Angle.createRadians(0), error: true };
-    if (this.isIdentity) {
-      result.error = false;
-      return result;
-    }
-    if (!this.isRigid())
-      return result;
-    const QMinusI = this.clone();
-    QMinusI.coffs[0] -= 1.0;
-    QMinusI.coffs[4] -= 1.0;
-    QMinusI.coffs[8] -= 1.0;
-    // Each column of (Q - I) is the motion of the corresponding axis vector
-    // during the rotation.
-    // Only one of the three axes can really be close to the rotation axis.
-    const delta0 = QMinusI.columnX();
-    const delta1 = QMinusI.columnY();
-    const delta2 = QMinusI.columnZ();
-    const cross01 = delta0.crossProduct(delta1);
-    const cross12 = delta1.crossProduct(delta2);
-    const cross20 = delta2.crossProduct(delta0);
-
-    const aa01 = cross01.magnitudeSquared();
-    const aa12 = cross12.magnitudeSquared();
-    const aa20 = cross20.magnitudeSquared();
-
-    const cross = cross01.clone(); // This will end up as the biggest cross product
-    const v0 = delta0.clone();  // This will end up as one of the two largest delta vectors
-    let aaMax = aa01;
-    if (aa12 > aaMax) {
-      cross.setFrom(cross12);
-      aaMax = aa12;
-      v0.setFrom(delta1);
-    }
-    if (aa20 > aaMax) {
-      cross.setFrom(cross20);
-      aaMax = aa20;
-      v0.setFrom(delta2);
-    }
-
-    if (aaMax === 0.0) {
-      // The vectors did not move.  Just accept the zero rotation, with error flag set.
-      return result;
+  /**
+   * Apply (in place) a jacobi update that zeros out this.at(i,j).
+   * @param i row index of zeroed member
+   * @param j column index of zeroed member
+   * @param k other row/column index (different from i and j)
+   * @param leftEigenVectors a matrix that its columns will be filled by eigenvectors of this Matrix3d
+   * (allocated by caller, computed and filled by this function)
+   */
+  private applyFastSymmetricJacobiUpdate(i: number, j: number, k: number, leftEigenVectors: Matrix3d): number {
```

Comment:
Do not revew this function. It just moved from bottom of the file here but I have not changed the content yet.